src/main/scala/akka/stream/alpakka/xlsx/Row.scala

```diff
@@ -17,7 +16,7 @@ final class Row(val rowIndex: Int, columnIndexToCell: TreeMap[Int, Cell]) {
     *         correspond to cells of adjacent columns. If a cell is missing in the `columnIndexToCell` mapping, a `Blank`
     *         cell is created with reference generated by `CellReference#generateReference`.
     */
-  def cells: Seq[Cell] = (1 to columnIndexToCell.keys.max).iterator
+  def cells: Seq[Cell] = (1 to Try(columnIndexToCell.keys.max).getOrElse(0)).iterator
```

Comment:
I guess you can rely on the ordering of keys since this is a `TreeMap`. `1 to columnIndexToCell.keys.lastOption.getOrElse(0)` seems to work well enough.